### PR TITLE
Implement uniqueNames() for issue #1779

### PR DESCRIPTION
**Pull request by TvdW**
I guess the code for issue 1779 would look something like this. :)

### CODE REVIEW

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -3665,6 +3665,28 @@ def removeEmptySeries(requestContext, seriesList):
...
+
+  .. code-block:: none
+
+    &target=uniqueNames(mostDeviant(server.*.disk_free,5),lowestCurrent(server.*.disk_free,5))
+
+  Draws servers with low disk space, and servers with highly deviant disk space, but never the same series twice.
+
+  """
+  newList = []
+  seenNames = {}
```
Comment (yati-sagade) [inline webapp/graphite/render/functions.py]: Can be a set (`seenNames = set()`) and then `seenNames.add(series.name)` :)

**Comment by obfuscurity** [comment]
I like this in general, but I would prefer to call it `uniqueSeries` or simply `unique`.

Should the function attempt to do any reconciliation of series with different contents (or determine preference)?

**Comment by obfuscurity** [comment]
refs #1779

**Comment by deniszh** [comment]
Hello @yati-sagade,
Would you mind to rename your function to `unique`? It will have nice match with `grep`, `exclude` etc.

**Comment by DanCech** [comment]
Closing in favor of #2063

**Pull request closed without merging**